doc/api/next_api_changes/behavior/19128-AG.rst

```diff
@@ -0,0 +1,7 @@
+Meta key specified name change on Qt5 backend
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *matplotlib.backends.backend_qt5.SPECIAL_KEYS* dictionary
+contains keys that do *not* return their unicode name
+instead they have manually specified names. The name
+for *QtCore.Qt.Key_Meta* has changed to ``'meta'`` to be consistent.
```

Comment:
```suggestion
for *QtCore.Qt.Key_Meta* has changed to ``'meta'`` to be consistent with the
other GUI backends.
```

Reply:
Can this instead be titled "harmonized key event data across backends" or something like that?  That way it can also note  the changes to `shift` labeling and the fixes to webagg.

I agree these changes are definitely a bug fix, I would bet that there exists more than one user with a non-US keyboard who discovered what keys _actually_ got sent when they hit "shift+2" and code against that (I am confident about this because I also use a non-qwerty layout and have some things configured in these double-negative ways) .  This will break them so we should note it, particularly because this is something where there is no (reasonable) way to do warnings or provide an easy way to get back to the old behavior.

Reply:
I haven't a lot of experience writing documentation nor api changes, also, English is not my main language. 

@tacaswell please can you take a look at my lastest commits and don't hesitate to do observations about typos/grammar, formatting, etc.